pkgs/applications/editors/emacs/elisp-packages/nongnu-devel-packages.nix

```diff
@@ -35,7 +35,23 @@ let
 
       super = imported;
 
-      overrides = { };
+      overrides = {
```

Comment:
Quite complex and tricky..
Future note: Should we dedup overrides between nongnu stable/devel?

Reply:
Hmm, the complexity is due to `src` being an unpacked tarball.  I would say the core of this workaround is simple:

```console
cp --verbose $content_directory/p4-16-mode-pkg.el $content_directory/p4-pkg.el
```

> Should we dedup overrides between nongnu stable/devel?

I think we should and it is tracked in https://github.com/NixOS/nixpkgs/issues/269550.